.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bb_drivers/package/sharp_drm/src"]
+	path = bb_drivers/package/sharp_drm/src
+	url = git@github.com:ardangelo/sharp-drm-driver.git
```

Comment:
You may want to create a central copy of this too on the same GitHub org, referenced this for now.